controllers/suite_test.go

```diff
@@ -116,3 +119,36 @@ var _ = AfterSuite(func() {
 		Expect(testEnv.Stop()).To(Succeed())
 	}
 })
+
+func ContainRefOfKind(kind string) types.GomegaMatcher {
```

Comment:
This should probably be ContainRefOfGroupKind and take in a group and a kind. (Also this is awesome, btw!)